### ISSUE TITLE
Strikethrough not working in v0.16.0

### ISSUE THREAD

**Issue by fyrefli7** [PHPOffice/PHPWord#1693]
### Describe the Bug

Strikethrough is no longer working in version 0.16.0, it is working in version 0.15.0

### Steps to Reproduce

```php
<?php
require __DIR__ . '/vendor/autoload.php';

$phpWord = new \PhpOffice\PhpWord\PhpWord();
$section = $phpWord->addSection();
$section->addText('Hello World!', array('strikethrough' => true));

$objWriter = \PhpOffice\PhpWord\IOFactory::createWriter($phpWord, 'Word2007');
$objWriter->save('/tmp/test.docx');
```
### Expected Behavior

In the above example 'Hello World!' should have the ~strikethrough~ formatting applied.  

Using "composer require phpoffice/phpword" to install the PHPWord results in strikethrough not being applied.  However using "composer require phpoffice/phpword:0.15.0" to install the PHPWord does apply strikethrough.

### Context
Ubuntu: 19.04
PHP Version: 7.2.19
LibreOffice: 6.2.5.2

**Comment by montypanday** [comment]
Can you please share more details. I could not replicate this issue in v0.16.0.

**Comment by liborm85** [comment]
Tested with version 0.17.0 and works correctly.

**Comment by NigelRel3** [comment]
Through a question on [Stackoverflow](https://stackoverflow.com/questions/67416303/how-can-i-make-phpword-to-cross-out-text-for-libreoffice), I managed to reproduce this with 

Ubuntu: 20.04
PHPOffice/PHPWord 0.18.1
LibreOffice: 6.4.7.2

After messing about with the block (in the document.xml)...

			<w:r>
				<w:rPr>
					<w:strike w:val="1" />
					<w:dstrike w:val="0" />
				</w:rPr>
				<w:t xml:space="preserve">Hello World!</w:t>
			</w:r>
I found that if you remove the `<w:dstrike w:val="0" />` element, the display works OK.

**Comment by shumiv** [comment]
I got the same problem. Solved it by downgrading to 0.15.0

**Comment by pvy** [comment]
maybe the function PhpWord/Style/AbstractStyle:: setPairedVal() should set the second value to null and not false
